src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return base * height / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: right triangle, area: %.2f sq. cm, base: %.2f cm, height: %.2f "
```

Comment:
To improve code readability, consider creating a variable for the area before the printf statement and then use it in the printf statement. This will separate the concerns of calculating the area and displaying the information.